wallet-new/server/Cardano/Wallet/API/V1/Handlers/Wallets.hs

```diff
@@ -71,8 +71,9 @@ deleteWallet
     :: (MonadWalletLogic ctx m)
     => WalletId
     -> m NoContent
-deleteWallet (WalletId walletId) =
-    V0.deleteWallet . V0.CId . V0.CHash $ walletId
+deleteWallet walletId = do
```

Comment:
or just
```haskell
deleteWallet = V0.deleteWallet <=< migrate
```
which is basic Monadic operation and we don't have to introduce 2 extra variables for this simple case.

Reply:
Valid suggestion, but I really don't like removing the parameter, because I find myself looking for it.

Reply:
I believe it is clear from types that there is a parameter here.

Reply:
Fine, I'll fix it, but I really don't like omitting parameters, types or no types.

Reply:
Well, omitting single parameter is usual scenario in haskell (point free) because there are combinators which are common for this use case (as function composition, and similar).

Omitting two or more paramaters make things harry and I avoid it as well